cfg/input.go

```diff
@@ -33,7 +34,7 @@ const (
 )
 
 var (
-	kafkaEmptyConsumerGroupError = errors.New("consumer_group can't be empty in kafka config")
+	kafkaEmptyConsumerGroupError = errors.New("consumer_group_id can't be empty in kafka config")
```

Comment:
why putting id ?
The id is more client id.

Reply:
In kafka documentation, it's defined as consumer group.id, so I kept their naming